tests/message-extensions/src/settings.html

```diff
@@ -0,0 +1,58 @@
+<!-- :snippet-start: message-ext-settings-page -->
+<html>
+<body>
+    <form>
+        <fieldset>
+            <legend>What would option would you like to search?</legend>
```

Comment:
Fix sentence?
Is there a possibility of making "Option A and B" more concrete examples of something someone would search?

Reply:
Like Teams AI TypeScript and Teams AI .NET or something?

Reply:
Could you also run lint on this file?

Reply:
update the prompt to "What's your favorite programming language" with "C#" and "Typescript as options. I also rant a linter.